tests/restheart.yml

```diff
@@ -0,0 +1,25 @@
+---
+runner: command
+dir: restheart
+args: [../restheart.sh]
+
+results:
+  postgresql:
+    stats:
+      # https://github.com/FerretDB/FerretDB/issues/2568
+      fail: 1
+    fail:
+      - restheart
+
+  sqlite:
+    stats:
+      # https://github.com/FerretDB/FerretDB/issues/2568
+      fail: 1
+    fail:
+      - restheart
+
+  mongodb:
+    stats:
+      fail: 1
```

Comment:
Ok, but why that configuration fail?

Reply:
It actually passes locally but fails on CI. I am still investigating.

Reply:
Then why was it submitted for review without any comment? "The test fails, I will just mark it as failing, that's fine, I guess"? No, it is not fine.

Reply:
Because there was tones of output and I couldn't identify the issue. Two transaction tests were failing due to replication not being configured.

Reply:
It will pass once #634 is merged as I have configured the MongoDB service as a single node replica set in that PR.

Reply:
done @AlekSi

Reply:
That issue is closed now

Reply:
indeed, will update that.

Reply:
Now we are back to my original question – why this configuration fails without any issue link? https://github.com/FerretDB/dance/blob/e26c12ec4b5718af0f8f7b3596798a6db8446456/README.md?plain=1#L14